cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,94 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should test pagination', () => {
+    cy.get('.left-pannel').click();
+  });
+
+  it('should test rows count selection', () => {
+    cy.get('.col-md-6').should('be.visible').click();
```

Comment:
The test 'should test rows count selection' is not implemented. Currently it just clicks and checks visibility of an element with the class name '.col-md-6'. Please implement the test according to the task description.